nodejs/week2/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,16 @@
+const cheapMeals = function (req, res) {
+  const mealsJson = require('../data/meals.json'); 
+  const reviewJson = require('../data/reviews.json');    
+  const cheapMealArr = mealsJson.filter((item) =>  {
+      return item.price < 50;
+    });
+  const cheapReview = mealsJson.map((cor) => {
+        return cor.review = reviewJson.filter((item) => {
+         if (cor.id === item.mealID) {

```

Comment:
1. The code can be written in a simpler way.
2. The callback has to return boolean, not undefined.